Add section on how to set up the Next.js SDK with the Vercel Edge Runtime (#5713)
getsentry/sentry-docs: src/platforms/javascript/common/troubleshooting/index.mdx

   plugins: [react()]
})
```

<PlatformSection supported={["javascript.nextjs"]}>

## Vercel Experimental Edge Runtime

> The Sentry Next.js SDK does not yet support the [Vercel Edge Runtime](https://vercel.com/docs/concepts/functions/edge-functions) which is used in Next.js middleware and Edge Routes when deployed on Vercel.
> If you want to use the Edge Runtime in combination with the Sentry SDK you need to manually opt the API routes that use the Edge Runtime out of Sentry instrumentation via the `excludedServersideEntrypoints` option in your `next.config.js`.
> 
> For example: Opting out the route `/api/posts` and all the routes starting with `/api/users/` will work like the following.
> 
> ```javascript {filename:next.config.js}
> const { withSentryConfig } = require("@sentry/nextjs");
> 
> const nextConfig = {
>   // Next.js configuration options...
> 
>   sentry: {
>     excludedServersideEntrypoints: [
>       "pages/api/posts",
>       /^pages\/api\/users\/.*/,
>     ],
>     // Other optional Sentry build-time configuration options...
>   },
> };
> 
> const sentryWebpackPluginOptions = {
>   // Sentry Webpack Plugin options...
> };
> 
> module.exports = withSentryConfig(nextConfig, sentryWebpackPluginOptions);
> ```
> 
> Please note that this will effectively disable the Sentry SDK for these API routes - removing all error and performance monitoring.

Reviewer: ```suggestion
The Sentry Next.js SDK doesn’t currently support [Vercel Edge Runtime](https://vercel.com/docs/concepts/functions/edge-functions) (which is used in Next.js middleware and Edge Routes when deployed on Vercel).
To use Edge Runtime in combination with the Sentry SDK, you'll need to manually opt out of the API routes that use the Edge Runtime in Sentry instrumentation via the `excludedServersideEntrypoints` option in your `next.config.js`.

For example: Opting out the route `/api/posts` and all the routes starting with `/api/users/` will work like this:

```javascript {filename:next.config.js}
const { withSentryConfig } = require("@sentry/nextjs");

const nextConfig = {
  // Next.js configuration options...

  sentry: {
    excludedServersideEntrypoints: [
      "pages/api/posts",
      /^pages\/api\/users\/.*/,
    ],
    // Other optional Sentry build-time configuration options...
  },
};

const sentryWebpackPluginOptions = {
  // Sentry Webpack Plugin options...
};

module.exports = withSentryConfig(nextConfig, sentryWebpackPluginOptions);
```

Please note, this will effectively disable the Sentry SDK for these API routes and remove all error and performance monitoring.
```
